tests/test_unit/test_reports.py

```diff
@@ -4,28 +4,136 @@
 
 
 @pytest.mark.parametrize(
-    "data_selection",
+    "valid_dataset",
     [
-        lambda ds: ds.position,  # Entire dataset
-        lambda ds: ds.position.sel(
-            individuals="ind1"
-        ),  # Missing "individuals" dim
-        lambda ds: ds.position.sel(
-            keypoints="key1"
-        ),  # Missing "keypoints" dim
-        lambda ds: ds.position.sel(
-            individuals="ind1", keypoints="key1"
-        ),  # Missing "individuals" and "keypoints" dims
+        "valid_poses_dataset",
+        "valid_bboxes_dataset",
+        "valid_poses_dataset_with_nan",
+        "valid_bboxes_dataset_with_nan",
     ],
 )
-def test_report_nan_values(
-    capsys, valid_poses_dataset_with_nan, data_selection
+@pytest.mark.parametrize(
+    "data_selection, list_expected_individuals_idcs",
```

Comment:
shoud `idcs` be `ids` or `idxs`? Feels like there is a stray `c`.

Reply:
I actually meant `idcs` as indices, plural of index.

I meant to distinguish it from IDs.

Will spell it out for clarity

Reply:
Ah, I typically use `idxs` for the same thing (and `idx` for singular). I had just never seen `idcs`